File: src/assets/CreateFile.java
package assets;
import java.io.BufferedReader;
import java.io.File;
import java.io.InputStreamReader;
public class CreateFile {
    // Function To Make New File
    public static void newFile()
    {
        String strPath , strName ;

        try {

            // Creating BufferedReadered object
            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
            System.out.print("Enter the file name : ");

            // Reading File name
            strName = br.readLine();
            System.out.print("Enter the file path : ");

            // Reading File Path
            strPath = br.readLine();

            // Creating File Object
            File file1 = new File(strPath + "\\" + strName + ".txt");

            // Method createNewFile() method creates blank file.
            file1.createNewFile();
        }

        catch (Exception ex1) {
            System.out.print("Failed to create a file.");
        }
    }
}

File: MapGenerator/assets/CreateFile.cs
using System;
using System.IO;

namespace Assets
{
    public class CreateFile
    {
        // Function To Make New File
        public static void NewFile()
        {
            string strPath, strName;

            try
            {
                // Reading File name
                strName = Console.ReadLine();
                // Reading File Path
                strPath = Console.ReadLine();

                // Creating File Object and creating blank file
                string fullPath = Path.Combine(strPath, strName + ".txt");
                using (FileStream fs = File.Create(fullPath))
                {
                }
            }
            catch (Exception)
            {
                Console.Write("Failed to create a file.");
            }
        }
    }
}